config/routes.rb

```diff
@@ -0,0 +1,5 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root to: 'pages#index'
+  resources :pages, only: [:index]
```

Comment:
I think you don't need this one.  Do you?

Reply:
@the-harry I need it as I have this `spec/requests/pages_spec.rb` file generated after running "rails g controller ..."